### DISCUSSION TITLE
Shutdown Button

### DISCUSSION THREAD

**Discussion by alimahran2612** [seerge/g-helper#3161] [Q&A]
Hi there, i have an issue with my power button it doesnt work at all or it needs like 50 click to work, and everytime i need to break it with my finger to work, so is anything can help me open the laptop after just opening the lid ? or just turn on the laptop with any other key i tried to look in the bios but i didnt find anything to help me in that !
my laptop is Asus Tuf gaming F15 
Asking for a fast reply, 
Thanks for your time 
Best Regards.!

**Comment by seerge** [comment]
@alimahran2612 hello, I don't know but looks like a broken button ? :) I would check up with the service

**Comment by alimahran2612** [comment]
any news ?

**Comment by alimahran2612** [comment]
i found out that its a common issue in that model , so if it can be turned on by any other method in bios as the other laptops it'd be amazing , or just once i lift the lid it opens
